config/puma.rb

```diff
@@ -11,7 +11,12 @@
 
 # Specifies the `port` that Puma will listen on to receive requests; default is 3000.
 #
-port        ENV.fetch('PORT') { 3000 }
+if ClowderCommonRuby::Config.clowder_enabled?
+  config = ClowderCommonRuby::Config.load
+  port = config.webPort
```

Comment:
Drop the `=`. The `port` is a method/function call.

Technically (I guess), you can put this to the default block:
```
port ENV.fetch('PORT') do
  if ClowderCommonRuby::Config.clowder_enabled?
    ClowderCommonRuby::Config.load.webPort
  else
    3000
  end
end
```